hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/shell/keys/DeleteKeyHandler.java

```diff
@@ -83,7 +78,7 @@ private void deleteFSOKey(OzoneBucket bucket, String keyName)
 
     // If Bucket layout is FSO and Trash is enabled
     // In this case during delete operation move key to trash
-    if (trashInterval > 0 && !skipTrash &&
+    if (trashInterval > 0 &&
```

Comment:
Correct me if I am wrong, if --skipTrash is passed here. One might know skipTrash is supported. Shouldn't we just throw an exception saying --skipTrash is not supported instead of removing the whole config?

Reply:
Since there will be no --skipTrash flag defined after this change, picocli should throw this exception for us stating that the flag is not recognized.

Reply:
Any option which is not supported for command it throws Unknown option. This is general behaviour for all commands.
If someone try to use skipTrash in key delete, it will throw 
`Unknown option: '--skipTrash'` and it provides usage details about the command.